addons/calendar/models/calendar_attendee.py

```diff
@@ -44,6 +46,15 @@ def _compute_common_name(self):
         for attendee in self:
             attendee.common_name = attendee.partner_id.name or attendee.email
 
+    @api.depends('state', 'event_id.attendee_ids', 'event_id.partner_id')
+    def _compute_is_alone(self):
+        for attendee in self:
+            all_declined = not any((attendee.event_id.attendee_ids - attendee).filtered(lambda a: a.state != 'declined'))
+            if len(attendee.event_id.attendee_ids) > 1 and attendee.event_id.partner_id.id == attendee.partner_id.id and all_declined:
+                attendee.is_alone = True
+            else:
+                attendee.is_alone = False
```

Comment:
ok, So if I understood well : we want to check if the organizer is the only attendee that has not declined the event. And we want to return true only if the current attendee is the organizer, otherwize, is_alone = False.

I would than rename the field into "is_organizer_alone" or "is_organizer_and_alone" or even "has_all_declined".
I summon @awa-odoo for renaming purpose !
So we avoid confusion with "is current user alone in that event".
Also, is_alone = False would suggest that your are not alone at that event even if you and all the other invited attendees declined as well, which makes no sense.
To make it more clear, i would rewrite a bit : 
```
for attendee in self:
    organizer = attendee.event_id.partner_id
    if organizer != attendee.partner_id:  # if attendee is not the organizer
        attendee.is_organizer_alone = False
        continue

    invited_attendees = attendee.event_id.attendee_ids - organizer
    all_declined = not any(event_attendees.filtered(lambda a: a.state != 'declined'))
    attendee.is_organizer_alone = invited_attendees and all_declined
```
which is a bit longer but easier to read.
But the end:
- either this method tells if the organizer is alone and we have to use that value with another test "if current attendee is the organizer" to modify the display of the event -> naming would be more like : 'is_organizer_alone" 
- either we integrate everything in the compute and naming would be -> "is_organizer_and_alone" or "has_all_declined" or something.

Or even put that on calendar event model and user the current user to check if he is the organizer of the event. 
I guess the objective is to display a warning to the user if his events are marked as "is_alone" ?

What do you think ?

I'm quite lost in fact.. :thinking:

Reply:
The goal is indeed to display an icon like google calendar next to the tilte of the event, in the case all attendees but the organizer have declined the event. I would not put it in calendar event model since when we display the events by attendee, the event is the same for all the attendees, it's just the color and popover of the event that are adapted with the info of the attendee.
So I think that it should still be in the calendar attendee model.

I agree for the renaming of the field, the two possibilities are pretty similar to me. I suppose it's more a question of what we prefer. I don't think we need to check if the current user is the organizer to display the icon on the event of the organizer, we can always display it, since it will only be display if we have the organizer checked in the filter.